package.json

```diff
@@ -18,7 +18,8 @@
     "check_version": "echo ${npm_package_version} $(npm view ${npm_package_name} version) | awk '{ if ($1 == $2) { exit 1 }}'",
     "dist": "npm-run-all clean build:tsc dist:sed-version",
     "dist:sed-version": "sed -i.bak -e \"s/exports.version = __VERSION__/exports.version = \\\"${npm_package_version}\\\"/\" build/src/core/version.js",
-    "lint": "npm run lint:tslint",
+    "lint": "npm run lint:tslint && npm run lint:eslint",
```

Comment:
you can use `npm-run-all` for this: `npm-run-all lint:tslint lint:eslint`

Reply:
oh yeah i guess we do already depend on it